duolingo.py

```diff
@@ -75,6 +75,37 @@ def get_activity_stream(self, before=None):
         except:
             raise Exception('Could not get activity stream')
 
+    def get_leaderboard(self, unit=None, before=time.time()):
```

Comment:
maybe int, float or str... the argument before in this case will be parsed in str in the line 91 as follow:

`url = url.format(unit, before)`

Anyway, to follow the pattern that is implemented in previous methods, I'll set before as a str in datetime format:

`    def get_leaderboard(self, unit=None, before=None):
        """
        Get user's rank in the week in descending order, stream from
        ``https://www.duolingo.com/friendships/leaderboard_activity?unit=week&_=time

        :param before: Datetime in format '2015-07-06 05:42:24'
        :param unit: maybe week or month
        :type before: str
        :type unit: str
        :rtype: List
        """
        if unit:
            url = 'https://www.duolingo.com/friendships/leaderboard_activity?unit={}&_={}'
        else:
            raise Exception('Needs unit as argument (week or month)')

        if before:
            url = url.format(unit, before)
        else:
            raise Exception('Needs str in Datetime format "%Y.%m.%d %H:%M:%S"')`